.github/workflows/ci.yml

```diff
@@ -12,14 +12,6 @@ concurrency:
   group: ${{ github.ref }}
   cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
 permissions: read-all
-env:
```

Comment:
Define environment variables here is meaningless unless we pass them as arguments to other CI files